### DISCUSSION TITLE
Configurable analog triggers

### DISCUSSION THREAD

**Discussion by mathieu4202** [cathery/sys-con#214] [Feature requests]
Hello,

It would be good to be able to set the analog triggers instead of the analog pads.
For example, set the right trigger instead of the up position of the right pad and the left trigger instead of the down position. 
This would allow in racing games to be able to define the analog triggers as accelerator and brake.
Thank you.

**Comment by amit27592** [comment]
I was able to do this for my Xbox360 controller by modifying `NormalizedButtonData Xbox360Controller::GetNormalizedButtonData()` inside Xbox360Controller.cpp.
Don't know enough to make the changes for all controllers though.

**Comment by mathieu4202** [comment]
I am not a developer, can you provide a download link to a ZIP of your modified sys-con folder?
Thank you

**Comment by amit27592** [comment]
Do you use an Xbox360Controller? or do you use some other controller?

**Comment by mathieu4202** [comment]
I use an Xbox360Controller.
